main/boost/APKBUILD

```diff
@@ -96,7 +96,7 @@ _set_options() {
 		runtime-link=shared
 		link=shared,static
 		cflags=-fno-strict-aliasing
-		--layout=system
+		--layout=tagged-1.66
```

Comment:
Why would we want to do this at this point? `--layout=system` makes things much simpler for us.